language/portuguese/form_validation_lang.php

```diff
@@ -10,31 +10,32 @@
+$lang['form_validation_alpha']                  = 'O campo {field} só pode conter caracteres do alfabeto.';
+$lang['form_validation_alpha_numeric']          = 'O campo {field} só pode conter caracteres alfanuméricos.';
+$lang['form_validation_alpha_numeric_spaces']   = 'O campo {field} só pode conter caracteres alfanuméricos e espaços.';
+$lang['form_validation_alpha_dash']             = 'O campo {field} só pode conter caracteres alfanuméricos, sublinhados (_) e traços (-).';
 $lang['form_validation_numeric']                = 'O campo {field} só pode conter números.';
-$lang['form_validation_is_numeric']             = 'O campo {field} só pode conter algarismos.';
-$lang['form_validation_integer']                = 'O campo {field} tem que conter um integer.';
-$lang['form_validation_regex_match']            = 'O campo {field} não está de acordo com o formato solicitado.';
-$lang['form_validation_matches']                = 'O campo {field} não é idêntico ao campo {param}.';
-$lang['form_validation_differs']                = 'O campo {field} tem que ser diferente do campo {param}.';
-$lang['form_validation_is_unique']              = 'O campo {field} tem que conter um valor único.';
-$lang['form_validation_is_natural']             = 'O campo {field} tem que conter um número natural.';
-$lang['form_validation_is_natural_no_zero']     = 'O campo {field} tem que conter umnúmero natural e diferente de zero.';
-$lang['form_validation_decimal']                = 'O campo {field} tem que conter um número decimal.';
-$lang['form_validation_less_than']              = 'O campo {field} tem que conter um número inferior a {param}.';
-$lang['form_validation_less_than_equal_to']     = 'O campo {field} tem que conter um número inferior ou igual a {param}.';
-$lang['form_validation_greater_than']           = 'O campo {field} tem que conter um número superior a {param}.';
-$lang['form_validation_greater_than_equal_to']  = 'O campo {field} tem que conter um número superior ou igual a {param}.';
-$lang['form_validation_error_message_not_set']  = 'Não existe uma mensagem de erro para o campo com o nome {field}.';
-$lang['form_validation_in_list']                = 'O campor {field} tem que ser de um dos seguintes tipos: {param}.';
+$lang['form_validation_is_numeric']             = 'O campo {field} só pode conter caracteres numéricos.';
+$lang['form_validation_integer']                = 'O campo {field} só pode conter um número inteiro.';
+$lang['form_validation_regex_match']            = 'O campo {field} não está no formato correcto.';
+$lang['form_validation_matches']                = 'O campo {field} não coincide com o campo {param}.';
+$lang['form_validation_differs']                = 'O campo {field} deve ser diferente do campo {param}.';
+$lang['form_validation_is_unique']              = 'O campo {field} deve conter um valor único.';
+$lang['form_validation_is_natural']             = 'O campo {field} deve conter um número inteiro positivo.';
```

Comment:
Changing to "inteiro positivo", gives the ideia that zero is not alowed. I think tha original "natural" is more explicit, and mathematical more accurate.

Reply:
I agree, but `inteiro positivo` is more understandable

Reply:
You should ask for a natural number. Not a positive int.